spec/lib/indexing/import_helper_spec.rb

```diff
@@ -3,27 +3,6 @@
 require 'rails_helper'
 
 RSpec.describe Indexing::ImportHelper do
-  describe "#display_filename" do
```

Comment:
These tests are now under spec/models/dataset_file_spec.rb since the functionality was moved to the DatasetFile class